### PR TITLE
[WIP] fix: Store tickets to Local

### LINKED ISSUE #1610: Store Present bought tickets Locally on device
**Describe the feature you'd like**
Due to unreliable internet connections , ticket doesn't get loaded . Present event tickets are important . So store them Locally on device and display ticket in TicketFragment . Remove the tickets from local storage , once event date has passed

Update : 
- EventBrite also does this . Display Tickets in Tickets_fragment (even if offline)
- No network calls for purchased Tickets
- I would also implement mechanism , such that if user re-install/ re-login in the app , it would get downloaded
That is 
First Check local -> if found then OK else store to local
- On logout , storage will be cleared.

**Would you like to work on the issue?**

- [ :heavy_check_mark:  ] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by theLimitBreaker**
Fixes #1610 
Changes: 
- [X] Made WorkFlow First Local -> Fetch Network(if not in Local) & store to Local  
- [X] Storing Events(Under Order), Tickets(Under Events), Attendees(Under Order) (For storing order)
- [ ] Storing Orders to display it locally

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -40,7 +40,7 @@ android {
             buildConfigField "String", "MAPBOX_KEY", '"'+MAPBOX_KEY+'"'
         }
         debug {
-            buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
+            buildConfigField "String", "DEFAULT_BASE_URL", '"https://api.eventyay.com/v1/"'
```
Comment (aggarwalpulkit596) [inline app/build.gradle]: undo this

Comment (theLimitBreaker) [reply app/build.gradle]: This will be changed in future PR. Dont worry

**Comment by theLimitBreaker** [comment]
Currently I have used `Logs` instead of Timber. But for final messages, It would be replaced by `Timber`

File: app/src/main/java/org/fossasia/openevent/general/attendees/ListAttendeeId.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package org.fossasia.openevent.general.attendees
+
+import com.github.jasminb.jsonapi.LongIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Type
+
+@Type("attendee")
+data class ListAttendeeId(
+    @Id(LongIdHandler::class)
```
Comment (aggarwalpulkit596) [inline app/src/main/java/org/fossasia/openevent/general/attendees/ListAttendeeId.kt]: why this? the list cannot be an Id especially of long type

Comment (theLimitBreaker) [reply app/src/main/java/org/fossasia/openevent/general/attendees/ListAttendeeId.kt]: This is not used anywhere till now. I am making changes to this currently.

**Comment by theLimitBreaker** [comment]
**Update**
- [X] Made WorkFlow First Local -> Fetch Network(if not in Local) & store to Local  
- [X] Storing Events(Under Order), Tickets(Under Events), Attendees(Under Order) (For storing order)
- [X] Storing Orders to display it locally
- [X] Showing attendee details offline

Work left
- [ ] code formatting

**Comment by anhanh11001** [comment]
@theLimitBreaker are you still working on this or may I fork this issue and finish it myself ?

**Comment by mariobehling** [comment]
@anhanh11001 Please finish it. Thanks!

**Comment by jogendra** [comment]
@theLimitBreaker Any update on this?

**Comment by theLimitBreaker** [comment]
I suppose @anhanh11001  started working on this

**Comment by anhanh11001** [comment]
yes this is already implemented #1761

**Pull request closed without merging**